volatility3/framework/automagic/linux.py

```diff
@@ -162,9 +326,12 @@ def find_aslr(
                 init_task.files.cast("long unsigned int")
                 - module.get_symbol("init_files").address
             )
-            kaslr_shift = init_task_address - cls.virtual_to_physical_address(
-                init_task_json_address
-            )
+            if layer_class == arm.AArch64:
```

Comment:
As the whole `find_aslr` function worked out of the box, without the need to use the `virtual_to_physical_address` function, I did this *quick fix* to avoid repeating the whole function (separating in `find_aslr_aarch64` and `find_aslr_intel`). 

But if in the future we have to add some other specific AArch64 code to `find_aslr` (e.g. handle some retrocompatibility), we will eventually do a clear separation. So I guess we could do it directly even if the code will be 99% the same ?